proxies/nodejs/handlers/location.ts

```diff
@@ -24,37 +25,56 @@ export const handleLocation = async (
             JSON.parse(body.params),
             data
         )
+        const lastParam = params[params.length - 1]
+
+        if (lastParam instanceof URL) {
```

Comment:
is there a reason to convert this to a URL object? seems unnecessary if you're just gonna use the url with `callbackURL.href`

Reply:
converting to URL since all the other values will be `< string | boolean | number >` so gives a better representation of the check